environments/payu/deploy.sh

```diff
@@ -1,7 +1,7 @@
 ### Update stable/unstable if necessary
-CURRENT_STABLE=$( get_aliased_module "${MODULE_NAME}"/analysis "${CONDA_MODULE_PATH}" )
+CURRENT_STABLE=$( get_aliased_module "${MODULE_NAME}"/payu "${CONDA_MODULE_PATH}" )
 NEXT_STABLE="${ENVIRONMENT}-${STABLE_VERSION}"
-CURRENT_UNSTABLE=$( get_aliased_module "${MODULE_NAME}"/analysis3-unstable "${CONDA_MODULE_PATH}" )
+CURRENT_UNSTABLE=$( get_aliased_module "${MODULE_NAME}"/payu-unstable "${CONDA_MODULE_PATH}" )
```

Comment:
Is this `payu-dev` rather than `payu-unstable`..?

Reply:
Good point. I might actually remove `payu-unstable` module alias as the separate `payu-dev` environment and module is a way to test the payu environment.